includes/Controllers/Item.php

```diff
@@ -862,6 +881,7 @@ public function get_player_data( $include_variations = false) {
 				'id'            => $this->model->id,
 				'originID'      => $this->post->ID,
 				'permalink'     => $this->get_permalink(),
+				'thumb'         => $this->get_thumbnail(),
```

Comment:
@tannerm is there a reason this was omitted before? I can't think of any reason it would be, but I'm also confused as to why it was left out previously.

Reply:
@jjroley looks like that was an oversight. Thanks for fixing!